Formula/kcov.rb

```diff
@@ -0,0 +1,37 @@
+class Kcov < Formula
+  desc "Code coverage tool for compiled programs, Python and Bash"
+  homepage "https://simonkagstrom.github.com/kcov/index.html"
+  url "https://github.com/SimonKagstrom/kcov/archive/v35.tar.gz"
+  sha256 "74c172dae2ac2866e0adc91d3fd80276e5acb970d11ac71679a0f7336897a476"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on :xcode # Needed for LLDB. See MachO work below.
+
+  def install
+    system "cmake", "-G", "Unix Makefiles", *std_cmake_args
+    system "make"
+
+    # As per notes in https://github.com/SimonKagstrom/kcov/issues/166
+    MachO::Tools.change_install_name(
+      "src/kcov",
+      "@rpath/LLDB.framework/LLDB",
+      MacOS::Xcode.bundle_path.join("Contents/SharedFrameworks/LLDB.framework/LLDB").to_s,
+    )
+
+    system "make", "install"
+  end
+
+  pour_bottle? do
+    reason "Need to set the XCode path explicitly"
```

Comment:
This is a problem; we don't like adding things that need to be built from source but can't be bottled. Can you elaborate on why this is necessary? It would be good to find a workaround for this.

Reply:
This is because of the XCode path. Because users may have different paths for XCode (apparently, I've got no idea how you do that, but it was raised as a previous issue), we need to do the rewrite step separately and apparently we can't just bottle everything before that.

Reply:
The rewrite could occur in the post_install method. Ideally we’d figure out a way with upstream that this isn’t required though.

Reply:
@palfrey FWIW, I think instead of hardcoding `/Applications/Xcode.app` you could check something like `ls -1 $(lldb -P)/../../LLDB`; that should work whether someone has Xcode, Xcode Beta or the CLT installed _as far as I'm aware_.

Doesn't resolve the issue in Homebrew sadly but might make life a bit easier for y'all upstream.

Reply:
Thinking about it more, it _might_ help on the Homebrew side actually. If you could link by default to the CLT `lldb` and that works as-expected you could then use:
```
  pour_bottle? do
    reason "The bottle needs the Xcode CLT to be installed."
    satisfy { MacOS::CLT.installed? }
  end
```
here, which Mike/Joe might find more palatable than disabling bottling entirely, especially since people are considerably more likely to have the CLT installed than Xcode given the size requirements of each.

Reply:
@DomT4 Done all of the above

Reply:
Apologies, I probably wasn't clear enough. I meant if you targeted the CLT by default upstream rather than Xcode that would be easier for Homebrew to work with.

Reply:
I suspect at this point that upstream will be mostly leaning towards the "why do you want to do that, we've fixed this already" opinion, which TBH is my thinking on the matter as well.